### DISCUSSION TITLE
Logging changes?

### DISCUSSION THREAD

**Discussion by ErikEngerd** [docker-mailserver/docker-mailserver#2979] [Q&A]
Hi,

Has something changed in the logging? Since the upgrade from 11.1.0 to the latest version, the container stdout no longer shows time stamps and the postfix logging for smtpd is no longer there.

Cheers
  Erik

**Comment by ErikEngerd** [comment]
I found the cause. The directory /var/log is owned by root and has permissions 0755. Changing that to 0777 and restarting postfix and rsyslogd fixes this issue.

**Comment by ErikEngerd** [comment]
The above change gave problems with logrotate. It turns out that the permissions 0755 (which were the same as the original) are ok  and the full logging is back with these settings after restarting the pod. So it is a bit unclear what caused logging to break after the update.

**Comment by casperklein** [comment]
Make sure, to always remove/recreate the container, instead of stop/start.

**Comment by ErikEngerd** [comment]
The container was restarted as part of the update. If I remember correctly, there was a + sign next to the permissions of /var/log in the ls -l output so perhaps additional access rights were defined (chacl) but it is impossible to verify now unless I check a backup which is time consuming. I will just do the next best thing for obscure issues, which is to look at them in detail when they occur a second time.
